config/matty-go.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "driver": "osgar.go:Go",
+          "in": ["emergency_stop", "pose2d"],
+          "out": ["desired_speed"],
+          "init": {
+            "max_speed": 0.2,
+            "dist": 1.0,
+            "timeout": 10
+          }
+      },
+      "platform": {
+        "driver": "osgar.platforms.matty:Matty",
+        "in": ["esp_data"],
+        "out": ["esp_data"],
+        "init": {}
+      },
+      "timer": {
+        "driver": "timer",
+        "in": [],
+        "out": ["tick"],
+        "init": {
+          "sleep": 0.1
+        }
+      },
+      "serial": {
+          "driver": "serial",
+          "in": ["raw"],
+          "out": ["raw"],
+          "init": {"port": "/dev/ttyUSB0", "speed": 115200}
+      }
+    },
+    "links": [
+      ["app.desired_speed", "platform.desired_steering"],
```

Comment:
I think we should strongly distinguish between the `deride_speed` and the `desired_steering`. The first one is speed and angular speed and the second one is speed and desired steering. Either the application will publish both, or (better) driver may support both `derired_speed` and `desired_steering`. The link above is confusing. I dont like it.

Reply:
OK, thanks for comment - in this case the 2nd number is 0 ... and means the same, but I understand/agree. I was actually going to change `go.py` (so I could change and test non-zero steering angle), but then I decided to have ver0 of Matty driver first. BTW how did you tested you Kloubak K2? Temporary modification of the code? Do you use angle or angular speed? (somehow the angle fits the platform better) thanks m.

Reply:
Yes, i was temporarily modifying the code during kloubak tests. I agree, the desired steering is better for some platforms. I think the go.py should publish only `desired_steering` because It is clearer for user. However, it also means that we have to add support for `desired_steering` in kloubak and eduro.